tests/chocolatey-tests/Hooks.Tests.ps1

```diff
@@ -0,0 +1,93 @@
+Import-Module helpers/common-helpers
+
+Describe "hooks tests" -Tag Chocolatey, Hooks {

```

Comment:
```suggestion
Describe "choco hooks tests" -Tag Chocolatey, HooksFeature {

```

Reply:
These make sense 😄 naming things is hard 😅